_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,161 @@
+
+This rule is designed specifically for [SC 1.4.5 Images of Text][sc1.4.5] which includes exceptions to the images it applies to. [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] does not consider any exceptions. Therefore, some images that are inapplicable for this rule can be applicable to [SC 1.4.9 Images of Text (No Exception)][sc1.4.9].
+
+- [Understanding Success Criterion 1.4.5: Images of Text][sc1.4.5]
+- [Understanding Success Criterion 1.4.9: Images of Text (No Exception)][sc1.4.9]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `img` element references an image resource that does not contain text.
+
+```html
+<img src="/test-assets/shared/fireworks.jpg" alt="fireworks going off behind the Eiffel tower at night" />
+```
+
+#### Passed Example 2
+
+This `img` element references an image resource that contains text but where the presentation of the text is essential to convey the information.
+
+```html
+<p>The following image illustrates the use of cursive writing in the late nineteenth century.</p>
+<img src="/test-assets/0va7u6/letter.jpg" alt="A letter written in 1894 showing the use of cursive writing" />
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `img` element references an image resource that contains text and the way the text is presented is not relevant.
+
+```html
+<img
+	src="/test-assets/0va7u6/textimage.jpg"
+	alt="The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test rules."
+/>
+```
+
+#### Failed Example 2
+
+This `input` element in the [Image Button][] references an image resource that contains text and the way the text is presented is not relevant.
+
+```html
+<input type="image" src="/test-assets/0va7u6/button.jpg" alt="Press me" />
+```
+
+#### Failed Example 3
+
+This `div` element has a `background-image` property that references an image resource that contains text and the way the text is presented is not relevant.
+
+```html
+<div style="background-image: url(/test-assets/0va7u6/textimage.jpg); width: 500px; height: 200px;" />
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This `img` element is not [visible][].
+
+```html
+<img
+	src="/test-assets/0va7u6/textimage.jpg"
+	alt="The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test rules."
+	style="display: none"
+/>
+```
+
+#### Inapplicable Example 2
+
+This `img` element is [marked as decorative][].
+
+```html
+<img src="/test-assets/0va7u6/textimage.jpg" alt="" />
+```
+
+#### Inapplicable Example 3
+
+This `img` element references an SVG document.
+
+```html
+<img src="/test-assets/shared/eu-logo.svg" alt="European Union flag" />
+```
+
+#### Inapplicable Example 4
```

Comment:
This doesn't seem necessary. If inapplicable 1 and 2 aren't picked up, this one won't be either.